### PR TITLE
Update domains

### PR DESCRIPTION
**Pull request by AntoineMille**

### CODE REVIEW

File: views/templates/hook/dashboard_zone_two.tpl [hunk 1]
```diff
@@ -25,19 +25,19 @@
 
 <section id="dashproducts" class="panel widget {if $allow_push} allow_push{/if}">
 	<header class="panel-heading">
-		<i class="icon-bar-chart"></i> {l s='Products and Sales' mod='dashproducts'}
+		<i class="icon-bar-chart"></i> {l s='Products and Sales' d='Modules.Dashproducts.Admin'}
 		<span class="panel-heading-action">
-			<a class="list-toolbar-btn" href="#" onclick="toggleDashConfig('dashproducts'); return false;" title="{l s='Configure' mod='dashproducts'}">
+			<a class="list-toolbar-btn" href="#" onclick="toggleDashConfig('dashproducts'); return false;" title="{l s='Configure' d='Modules.Dashproducts.Admin'}">
```
Comment (AlexEven) [inline views/templates/hook/dashboard_zone_two.tpl]: Admin.Actions

File: views/templates/hook/dashboard_zone_two.tpl [hunk 2]
```diff
@@ -25,19 +25,19 @@
...
 				<i class="process-icon-configure"></i>
 			</a>
-			<a class="list-toolbar-btn" href="#"  onclick="refreshDashboard('dashproducts'); return false;"  title="{l s='Refresh' mod='dashproducts'}">
+			<a class="list-toolbar-btn" href="#"  onclick="refreshDashboard('dashproducts'); return false;"  title="{l s='Refresh' d='Modules.Dashproducts.Admin'}">
```
Comment (AlexEven) [inline views/templates/hook/dashboard_zone_two.tpl]: Admin.Actions

File: views/templates/hook/dashboard_zone_two.tpl [hunk 3]
```diff
@@ -25,19 +25,19 @@
...
 				<i class="process-icon-refresh"></i>
 			</a>
 		</span>
 	</header>
 
 	<section id="dashproducts_config" class="dash_config hide">
-		<header><i class="icon-wrench"></i> {l s='Configuration' mod='dashproducts'}</header>
+		<header><i class="icon-wrench"></i> {l s='Configuration' d='Modules.Dashproducts.Admin'}</header>
```
Comment (AlexEven) [inline views/templates/hook/dashboard_zone_two.tpl]: Admin.Global

**Comment by maximebiloe** [comment]
Thank you @AntoineMille

**Pull request merged**